src/modules/commander/commander.cpp

```diff
@@ -1731,6 +1733,9 @@ int commander_thread_main(int argc, char *argv[])
 									(hrt_abstime)disarm_when_landed * 1000000);
 			}
 
+			int32_t v;
+			param_get(_param_acro_att, &v);
+			acro_att = (v != 0);
```

Comment:
@LorenzMeier cool! I didn't know about @boolean and couldn't find it in code before you mentioned it